pkg/kudoctl/packages/types.go

```diff
@@ -43,9 +64,152 @@ func (p Parameter) Less(x, y int) bool {
 	return p[x].Name < p[y].Name
 }
 
+// Templates is a map of file names and stringified files in the template folder of an operator
```

Comment:
Upon thinking about it: all of the new code below belong into a separate folder e.g. `/packages/verify/*`. The types could then be placed in a new `packages/verify/types.go` and the methods could live in `/packages/verify/templates/templates.go`. This way verify templates would be addressed with package notation `verify.Templates` which helps avoid the confusion with other, similar named types. 
This might look like overkill at first glance but is actually very well within [Go best practicies](https://www.youtube.com/watch?v=MzTcsI6tn-0). It also helps a lot when dealing with circular dependencies.

Reply:
Upon more thinking: the new `/verify` package **does not** have to leave underneath `/packages` at all.